doc/btp_spec.txt

```diff
@@ -1381,6 +1383,10 @@ Commands and responses:
 		Connected Event (or Disconnected Event in case of an error)
 		shall be expected after issuing this command.
 
+		if Num > 1 then Enhanced Credit Based L2CAP channel shall be created.
```

Comment:
I don't get why the 'num' parameter is needed. You could issue this command num times to get expected number of channels. Anyway, the 'Num > 1' for  Enhanced Credit Based L2CAP seems to be a hack. I would rather add "mode" parameter to specify whether it's LE Credit Based Flow Control Mode or Enhanced Credit Based Flow Control Mode.